src/net_processing.h

```diff
@@ -69,6 +70,6 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
 };
 
 /** Relay transaction to every node */
-void RelayTransaction(const uint256& txid, const uint256& wtxid, const CConnman& connman) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+void RelayTransaction(const uint256& txid, const uint256& wtxid, const CConnman& connman, bool flood) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
Consider making `flood=false` the default, so that call sites don't need to be updated for this change.